api/src/main/java/org/openmrs/OrderAttributeType.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import org.openmrs.attribute.AttributeType;
+import org.openmrs.attribute.BaseAttributeType;
+
+/**
+ * A user-defined extension to the {@link Order} class.
```

Comment:
You want to make this comment more clear to the final user of the docs.

Reply:
@dkayiwa and @tendomart thanks let me look into those and make another push. But I do not seem to get the cause of the failures.